main.py

```diff
@@ -25,8 +26,9 @@ def generate_subspectrogram():
 
     # building the model
     from models.cnn import ConvModel as Model
+    # from models.pianocktail_gru import PianocktailGRU as Model
```

Comment:
It might be worth adding a config variable or a string parameter to easily change the model trained